packages/jetpack-logo/tests/fixtures.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$current = dirname( __FILE__ );
+define( 'WP_CONTENT_DIR', realpath ( $current . '/../../../../..' ) );
```

Comment:
This feels pretty fragile. Any other suggestions? In any case I'd expect to see something like this in a bootstrap instead of in a fixture file.